src/main/java/codeu/controller/BotController.java

```diff
@@ -51,7 +51,10 @@ void setUserStore(UserStore userStore) {
   public void registerBot(Bot bot) {
     botMap.put(bot.getMentionKey().toLowerCase(), bot);
 
-    userStore.addUser((User)bot);
+    // First time adding to the UserStore
```

Comment:
To me this makes sense to be in the `UserStore` class, since you should never add two users with the same id. Not a hard comment though, so I'm fine if you want to keep it here.